controllers/storagecluster/volumegroupsnapshotterclasses.go

```diff
@@ -0,0 +1,175 @@
+package storagecluster
+
+import (
+	"fmt"
+	"reflect"
+
+	groupsnapapi "github.com/kubernetes-csi/external-snapshotter/client/v8/apis/volumegroupsnapshot/v1beta1"
+	snapapi "github.com/kubernetes-csi/external-snapshotter/client/v8/apis/volumesnapshot/v1"
+	ocsv1 "github.com/red-hat-storage/ocs-operator/api/v4/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/klog/v2"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+type GroupSnapshotterType string
```

Comment:
no need to export this type as its not used outside?

Reply:
@Madhu-1 currently no but this would mean changing all the variable types from GroupSnapshotterType to string directly in functions such as newVolumeGroupSnapshotClass and in generate.go file as well. Is this the expected format?

Reply:
```suggestion
type groupSnapshotterType string
```

can we do something like this as its used outside of this file?